MapboxMobileEvents/MMEEventsManager.m

```diff
@@ -273,19 +290,21 @@ - (void)createAndPushEventBasedOnName:(NSString *)name attributes:(NSDictionary
     }
     
     if (event) {
-        [self pushDebugEventWithAttributes:@{MMEEventKeyLocalDebugDescription: [NSString stringWithFormat:@"Pushing event: %@", event]}];
+        [self pushDebugEventWithAttributes:@{MMEDebugEventType: MMEDebugEventTypePush,
```

Comment:
This is fine for now but for MMEDebugEventTypePush and MMEDebugEventTypeTurnstile similarly, we should probably make event sub-types to make it easier to visualize them when we build something to parse these logs. Otherwise MMEDebugEventTypePush success and failure case will be hard to discern unless we parse the MMEEventKeyLocalDebugDescription